src/components/Manage/Cascader.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Cascader } from 'antd'
+
+const options = [{
```

Comment:
The options of airports could be fetched according to [Get IATA List](https://uranus.docs.apiary.io/#/reference/flight/airports/get-iata-list?mc=reference%2Fflight%2Fairports%2Fget-iata-list%2F200).